erdpy/validators/validators.py

```diff
@@ -1,13 +1,21 @@
 import binascii
 import logging
 
+from erdpy import config
 from erdpy.accounts import Address
 
 logger = logging.getLogger("validators")
 
 _STAKE_SMART_CONTRACT_ADDRESS = "erd1qqqqqqqqqqqqqqqpqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqplllst77y4l"
 
 
+def estimate_gas_limit(args, num_keys):
+    num_bytes = len(args.data)
+    gas_limit = config.MIN_GAS_LIMIT + num_bytes * config.GAS_PER_DATA_BYTE
```

Comment:
Is this correct? Do we need to add the `config.MIN_GAS_LIMIT`? I mean, usually it makes sense, but since `num_keys >= 1`, the gas limit is over the minimum gas limit anyway.

Reply:
We have to use MIN_GAS_LIMIT every time, this is the formula for gas used by a system smart contract call